app/assets/stylesheets/application.scss

```diff
@@ -225,6 +225,74 @@ main {
   font-size: 15px;
 }
 
+.nav-tabs .nav-link {
+  border: none;
+}
+
+.nav-tabs .nav-link.active {
+  background-color: transparent;
+  font-weight: bold;
+  border-bottom: 3px solid #696df2;
+}
+
+.bg-purple {
+  background-color: rgb(226, 225, 253);
+}
+
+.system-wrapper {
+  display: grid;
+  grid-template-columns: 3fr 4fr 4fr 2fr 1fr;
+  grid-gap: 1em;
+  grid-auto-rows: minmax(100px, auto);
+}
+
+.system-wrapper img {
+  width: 100%;
+  height: 100%;
+}
+
+.system-box:nth-child(1) {
+  border-bottom: 1px solid #ccc;
+}
+
+.system-box {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-auto-rows: minmax(100px, auto);
+  justify-content: center;
+}
+
+.system-box-title{
+  font-size: 15px;
+}
+
+.system-box-five {
+  position: absolute;
+  top: 10%;
+  right: 2%;
+}
+
+.status-on {
+  color: rgb(46, 170, 45);
+}
+
+.status-off {
+  color: red;
+}
+
+.position-relative {
+  position: relative;
+}
+
+.card-footer button {
```

Comment:
remove all  of these. We can achieve the same result using bootstrap alone with the exception of bg color. For that, just go to the .btn-outline-purple class which you defined earlier and set the background color to white instead of transparent. Everything else can be achieved using bootstrap alone. Replace the markup with this:
```html
<div class="card-footer position-relative bg-purple">
<div class="row align-items-center">
    <div class="col text-muted">
    <span >Zone A</span><span class="ml-5">ODU# 1</span>
    </div>
    <div class="col text-right">
      <button type="button" class="btn btn-outline-purple px-3">Adjust</button>
    </div>
  </div>
  </div>
```